JU Quiz/QuestionViewController.swift

```diff
@@ -16,11 +16,22 @@ class QuestionViewController: UIViewController {
     @IBOutlet weak var buttonAnswerD: UIButton!
     
     private var haveWon = false
+    var questions: [Question] = [] {
+        didSet {
+            question = questions.removeFirst()
+        }
+    }
     var question: Question?
+    var numberOfQuestions = 0
+    var rightAnswers = 0
```

Comment:
You used `numberOfQuestions` as a property name and here you could use `numberOfRightAnwers`.

Reply:
```suggestion
    var numberOfRightAnswers = 0
```

Reply:
Yes, that could be changed, but we will change this part later on, when we implement CoreData.